### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by megaoka**
Proposed grammatical fixes and changes to item names and descriptions.

Please let me know if these are helpful, I would be happy to do more.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -3861,30 +3861,30 @@ ITEM_20150323_004190	Unidentified dagger
...
 ITEM_20150323_004237	Monster's sample requested by Researcher Gares
-ITEM_20150323_004238	Tough Stem of vermilion Sharkmolly
+ITEM_20150323_004238	Vermilion Sharkmolly's Tough Stem
 ITEM_20150323_004239	Heart of Pink Chuparuka
 ITEM_20150323_004240	Monster's sample requested by Pyromancer
-ITEM_20150323_004241	Research material of Pyromancer
+ITEM_20150323_004241	Pyromancer's research material
 ITEM_20150323_004243	Leather carved with magic circle
-ITEM_20150323_004245	A drug gave from Master Hunter
+ITEM_20150323_004245	A drug obtained from the Master Hunter
```
Comment (Sunaries) [inline ITEM_en.tsv]: Master Hunter -> Hunter Master

File: ITEM_en.tsv [hunk 2]
```diff
@@ -4053,55 +4053,55 @@ ITEM_20150401_004415	Superior Infantry Gauntlets
...
-ITEM_20150401_004454	Superior Magic Bangle
-ITEM_20150401_004455	Superior Accuracy Bangle
+ITEM_20150401_004447	Superior Key Pendant
+ITEM_20150401_004448	Sage's Necklace
+ITEM_20150401_004449	Light Ronesa's Bangle
+ITEM_20150401_004450	Superior Ronesa's Bangle
+ITEM_20150401_004451	Ronesa's Ring Bracelet
+ITEM_20150401_004452	Ronesa's Noble Bracelet
+ITEM_20150401_004453	Superior Bangle of Health
+ITEM_20150401_004454	Superior Magical Bangle
```
Comment (Sunaries) [inline ITEM_en.tsv:4091]: Superior Magical Bangle -> Superior Bangle of Magi
I don't know what this item does, but the Korean string tells it is Magi.

**Comment by Sunaries** [comment]
You have added an **'s** to a lot of strings, but not all of them. Why? Ex) Superior Grima set, Superior Infantry etc.
In my opinion they sound perfectly fine without **'s**, but that's just my two cents.

File: ITEM_en.tsv [hunk 3]
```diff
@@ -4017,34 +4017,34 @@ ITEM_20150401_004379	Fedimian Gauntlets
...
 ITEM_20150401_004405	Infantry Leggings
 ITEM_20150401_004406	Infantry Armor
-ITEM_20150401_004407	Arch Mage Gloves
-ITEM_20150401_004408	Arch Mage Boots
-ITEM_20150401_004409	Arch Mage Pants
-ITEM_20150401_004410	Arch Mage Robe
+ITEM_20150401_004407	Arch Mage's Gloves
+ITEM_20150401_004408	Arch Mage's Boots
+ITEM_20150401_004409	Arch Mage's Pants
+ITEM_20150401_004410	Arch Mage's Robes
```
Comment (Sunaries) [inline ITEM_en.tsv]: Robes -> Robe

File: ITEM_en.tsv [hunk 4]
```diff
@@ -4017,34 +4017,34 @@ ITEM_20150401_004379	Fedimian Gauntlets
...
-ITEM_20150401_004401	Skirmisher Pants
-ITEM_20150401_004402	Skirmisher Tunic
+ITEM_20150401_004391	Guard's Gauntlets
+ITEM_20150401_004392	Guard's Greaves
+ITEM_20150401_004393	Guard's Leggings
+ITEM_20150401_004394	Guard's Armor
+ITEM_20150401_004395	Mage's Gloves
+ITEM_20150401_004396	Mage's Boots
+ITEM_20150401_004397	Mage's Pants
+ITEM_20150401_004398	Mage's Robes
```
Comment (Sunaries) [inline ITEM_en.tsv]: Robes -> Robe

File: ITEM_en.tsv [hunk 5]
```diff
@@ -4017,34 +4017,34 @@ ITEM_20150401_004379	Fedimian Gauntlets
...
 ITEM_20150401_004381	Fedimian Leggings
 ITEM_20150401_004382	Fedimian Armor
-ITEM_20150401_004383	Magician Gloves
-ITEM_20150401_004384	Magician Boots
-ITEM_20150401_004385	Magician Pants
-ITEM_20150401_004386	Magician Robe
+ITEM_20150401_004383	Magician's Gloves
+ITEM_20150401_004384	Magician's Boots
+ITEM_20150401_004385	Magician's Pants
+ITEM_20150401_004386	Magician's Robes
```
Comment (Sunaries) [inline ITEM_en.tsv]: Robes -> Robe

**Comment by megaoka** [comment]
Thanks for the changes, yeah some seem a little redundant but some of them sounded off without the possessive apostrophe.

**Comment by megaoka** [comment]
Thanks for the changes, yeah some seem a little redundant but some of them sounded off without the possessive apostrophe.

**Comment by Sunaries** [comment]
@megaoka I can't change anything in your own pull request. That is something you have to do. :) 
All I did was leave notes on particular lines that you changed.

File: ITEM_en.tsv (hunk 1 above)
Comment (megaoka) [inline ITEM_en.tsv]: Changed the words around.

**Comment by megaoka** [comment]
Gotcha. Still learning GitHub. I changed the apostrophes on the nonspecific nouns, but still felt appropriate on specific nouns.

**Comment by Sunaries** [comment]
@megaoka No problem. I was new here too and I had to get used to it as well. You forgot to change "Master Hunter" to "Hunter Master". Good job by the way. :+1:

**Comment by megaoka** [comment]
I just did replace all and went over changes. Everything checks out.

Thank you. Glad to be of help.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**